src/Symfony/Bridge/Twig/composer.json

```diff
@@ -17,7 +17,6 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "symfony/security-csrf": "~2.6|~3.0.0",
```

Comment:
In ``require-dev`` section is  ``symfony/security`` which replaces ``symfony/security-csrf``. The ``symfony/security-csrf`` is optional dependency for Form component, see https://packagist.org/packages/symfony/form.